### ISSUE TITLE
WinMLRunner.exe. Evaluating  ..[FAILED]

### ISSUE THREAD

**Issue by yahuuu** [microsoft/Windows-Machine-Learning#379]
I convert the pt model to the onnx model and verify that the onnx model has good output accuracy. But my onnx model cannot evaluateing with GPU

error like:
====
Binding (device = CPU, iteration = 9, inputBinding = CPU, inputDataType = Tensor, deviceCreationLocation = WinML)...[SUCCESS]
Evaluating (device = CPU, iteration = 9, inputBinding = CPU, inputDataType = Tensor, deviceCreationLocation = WinML)...[SUCCESS]
...
etc
...
Binding (device = GPU, iteration = 1, inputBinding = CPU, inputDataType = Tensor, deviceCreationLocation = WinML)...[SUCCESS]
**_[FAILED]
Evaluating (device = GPU, iteration = 1, inputBinding = CPU, inputDataType = Tensor, deviceCreationLocation = WinML)...[FAILED]_**
====

command:
WinMLRunner.exe  -model D:\projects\PYTHON\u2net_matting_verify\u2net_matting_verify\utils\test.onnx -iterations 9 -perf -GPU

environment:
WinMLRunner.exe  v1.0.1.0   x64 executable 
win10 20H2 19042.685, Intel(R) Core(TM) i7-8700 CPU, RAM 32.0 GB  x64
GPU:  Intel UHD Graphics 630   GPU Driver Version: 25.20.100.6473

I tran

**Comment by ryanlai2** [comment]
Hi @yahuuu, thanks for reaching out. May I ask if you can share your model with us for reproduction purposes?

**Comment by ryanlai2** [comment]
Can you also share your ONNX operator set version of the model? Are you using inbox or NuGet  version of WinML?

**Comment by yahuuu** [comment]
Hi @ryanlai2,  This is the pytorch [model](https://drive.google.com/file/d/1Nf1ZxeJZJL8Qx9KadcYYyEmmlKhTADxX/view) I'm using.  
This is a pre-trained model released by the [MODNet community](https://github.com/ZHKKKe/MODNet/tree/master/demo/video_matting/custom).
I referenced the official [ONNX](https://github.com/onnx/tutorials/blob/master/tutorials/PytorchCaffe2SuperResolution.ipynb) conversion code, pytorch -> onnx.

version : )
pytorch==1.7.0+cpu
torchvision==0.8.0
WinMLRunner.exe v1.0.1.0 && v1.2.2  x64

code : )
“”“
    torch.onnx.export(modnet,
                      example,
                      export_onnx_file,
                      opset_version = 10,
                      export_params=True,
                      do_constant_folding = True,
                      input_names = ["input"], 
                      output_names = ["output"],  
                      dynamic_axes = {"input": {0: "batch_size"},
                                      "output": {0: "batch_size"} })
”“”

**Comment by yahuuu** [comment]
opset_version = 12,
Waiting for your answer.
